JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -80,6 +80,18 @@
         }
 
         /**
+        * Checks if error has no meaningful data inside. Ususally such errors are received by window.onerror when error
+        * happens in a script from other domain (cross origin, CORS).
+        */
+        public static isMeaninglessError(message: string, url: string, lineNumber: number, columnNumber: number, error: Error): boolean {
+            return message == "Script error."
```

Comment:
Is "Script error" not browser specific? Also what about localization? Is there any way we can check for it without checking for text?